### ISSUE TITLE
Stuck at boot when installing Big Sur

### ISSUE THREAD

**Issue by opatotto** [EETagent/T480-OpenCore-Hackintosh#62]
Hi, I'm having some issues at the time my installation process is running, after Big Sur is downloaded (10 min to finish), my thinkpad reboots and can't pass this screen and keeps rebooting, also I followed the UEFI settings:

![20201122_233201](https://user-images.githubusercontent.com/43834144/100018218-c00d1700-2da1-11eb-9421-81a7871686f4.jpg)

I'm gonna try some fixes on dortania guide, if you guys know about this issue I appreciate it

**Comment by EETagent** [comment]
What is your SSD?

**Comment by opatotto** [comment]
Samsung Evo 970 Plus SSD 500GB... I have a different thinkpad model T480 i7 8650U + Nvidia 150mx, I have 2 SSD first one Windows and Linux on stock WWAN slot, second one is Evo 970 Plus, I have modified some options on the SecureBoot-Misc section, tried to disable it, also with your SMBIOS, it doesn't work ... I think is related with secure boot.

**Comment by Charlyo** [comment]
That SSD gives problems in MacOS. Have you updated the firmware to its latest version?

**Comment by opatotto** [comment]
Yes, latest version on Samsung Magician, Is it possible to update from Catalina to Big Sur without problems? Maybe with Catalina I have a chance

**Comment by EETagent** [comment]
Weird, do not know how MBP15,2 vs 14,1 difference should affects booting. SecureBootModel is currently set to j132 (MBP 15,2), but that does not cause incompatibilities when using MBP14,1 SMBIOS.

Are you using master? Current release version is not compatible with latest Big Sur since Beta 10. 

In both cases, try YogaSMC branch. I am gonna merge it soon.
https://github.com/EETagent/T480-OpenCore-Hackintosh/tree/yogasmc

**Comment by opatotto** [comment]
Yes I noticed the j132 was for MBP15,2... And made some changes on that section but didn't work, I made my usb with Catalina and then updated with Big Sur, right now is working but I need to remap my usb ports, I think this will be a solution.

**Comment by EETagent** [comment]
Any progress on this front, did you tried with latest OpenCore

**Comment by opatotto** [comment]
I tried with OP 0.6.4, I haven't test it yet with 0.6.5
